### ISSUE TITLE
How to get started

### ISSUE THREAD

**Issue by gennifer** [dpborland/RLTR#1]
can you post up a readme with terminal commands step by step on how to get this up and running?

ie for my current react project:
# Install
git clone git@github.com:MidwestAccessCoalition/react-drive-site.git
^^^ or use the HTTPS version if you're not using SSH key
cd react-drive-site
npm install
# Usage
npm run build
Open new session
npm run start

**Comment by dpborland** [comment]
https://hackernoon.com/setting-up-a-react-project-from-scratch-6cb7cbb4da28

**Comment by dpborland** [comment]
If that's not what you're after just let me know

**Comment by gennifer** [comment]
yeah was just looking for the simple commands to get this particular repo started with all package installs and build and server
